packages/params/src/forkName.ts

```diff
@@ -112,10 +112,10 @@ export function isForkPostFulu(fork: ForkName): fork is ForkPostFulu {
 
 export type ForkPreBlobs = ForkPreDeneb;
 export type ForkPostBlobs = ForkPostFulu;
-export type ForkBlobs = ForkName.deneb | ForkName.electra;
-export const forkBlobs = [ForkName.deneb, ForkName.electra];
+export type ForkBlobs = ForkName.deneb | ForkName.electra | ForkName.fulu;
```

Comment:
why do we still have this, I thought we got rid of it in https://github.com/ChainSafe/lodestar/pull/7441 cc @matthewkeil

Reply:
The `peerDAS` branch doesn't have that change yet, so this will need to be updated after `unstable` merges into `peerDAS`